packages/xo-server/src/index.js

```diff
@@ -45,6 +47,10 @@ import { configure } from '@xen-orchestra/log/configure'
 
 // ===================================================================
 
+authenticator.options = { crypto }
```

Comment:
It's weird to globally mutate a module (IMHO a bad practice), but I can see it's in the official doc of this package.

Maybe we should had a comment/link to highlight this?